### PR TITLE
Mod/11664 county fips api

### PR DESCRIPTION
**Pull request by jagu65**
**High level description:**
Autocomplete now using fips data from the autocomplete/location api, instead of the local json files. Local json files deleted.

**JIRA Ticket:**
[DEV-11664](https://federal-spending-transparency.atlassian.net/browse/DEV-11664)

**Mockup:**
n/a

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [ ] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [ ] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/js/models/v2/search/LocationEntity.js [hunk 1]
```diff
@@ -40,7 +40,8 @@ const LocationEntity = {
             city_name: item.city_name || '--',
             zip_code: item.zip_code || '--',
             original_cd: item.original_cd || '--',
-            current_cd: item.current_cd || '--'
+            current_cd: item.current_cd || '--',
+            county_fips: item.county_fips || '--'
```
Comment (anjenkin) [inline src/js/models/v2/search/LocationEntity.js:44]: you could slice(2) directly on the data that we get from the api.  i can't think of anywhere we would need the fips code to include the additional two characters in the front

Comment (jagu65) [reply src/js/models/v2/search/LocationEntity.js]: Fixed and added state fips as well.

**Comment by nick-torres** [comment]
were the files deleted?

**Comment by jagu65** [comment]
> were the files deleted?

The county json files are still needed for the congressional district feature, for now.

**Review by nick-torres — APPROVED** [review]

**Pull request merged**